src/portals/hope/features/claims/claim-details/ClaimCounterparty/InvoiceModal.tsx

```diff
@@ -48,31 +54,154 @@ gql`
       content
     }
   }
+
+  mutation settleRecoveryInvoice(
+    $recoveryInvoiceId: ID!
+    $request: SettleRecoveryInvoiceInput!
+  ) {
+    recoveryInvoice_settle(
+      recoveryInvoiceId: $recoveryInvoiceId
+      request: $request
+    ) {
+      id
+      isSettled
+    }
```

Comment:
I think there might be some more data points here that could be could to incude
```suggestion
    recoveryInvoice_settle(
      recoveryInvoiceId: $recoveryInvoiceId
      request: $request
    ) {
      id
      isSettled
      settledBy
      settledAmount
      ...
    }
```